assets/styles/code-title.scss

```diff
@@ -0,0 +1,14 @@
+.code-title {
+  color: rgba(238, 130, 238, 0.87);
+  background-color: rgba(68, 74, 90, 0.53);
+  font-family: monospace;
+  display: inline-block;
+  position: relative;
+  padding: 0.3em 0.6em 0.6em;
+  margin-bottom: -0.3em;
+  z-index: -1;
+  paint-order: fill;
+  border-top-left-radius: 0.3em;
+  border-top-right-radius: 0.3em;
+  font-size: 0.9em;
+}
```

Comment:
probably a good idea to define overflow rules for if the title is too long for small screens

Reply:
![code-title-issue](https://user-images.githubusercontent.com/26066340/176347967-3e52fe30-9aaf-4b08-a9cb-7b26b64cdf89.png)

There's something I don't know how to resolve... indented code block titles seems to have strange padding/margin compared to the unindented variants. The `overflow: hidden;` rule then makes this worse, and it then floats, as seen in the image. 

Any suggestions 😞? I will try look into this later today/tomorrow.

Reply:
I wouldn't merge this just yet. Two things I've noticed:

- Sometimes the code block title is rendered twice when navigating back and forth (there will be two side-by-side). 
- With `enableSPA` set to false, no code blocks are rendered.

This doesn't happen on my site, however I'm on an older commit of Quartz 3.2 which still has the `draw` function instead of `render` in the `head.html` partial. Furthermore, the solution I'm incorporating here is different as I believe it to be "cleaner". The issues could be a result of either of these differences.

I'll look into it later today/tomorrow. Any suggestions/thoughts are welcome 👍🏻

Reply:
@aidenybai could you leave a review as well? i think you are probably more familiar with these internals than me

Reply:
The overflow issue, plus the different spacing of the code-title when indented is fixed. I found there was a `ol` selector which changed line height, I overrode it with a consistent `1 em` for all code titles.

Reply:
The code titles are consistently working, with SPA and without, as of the commit I just pushed.